src/care/LoopFuser.h

```diff
@@ -1341,7 +1351,7 @@ void LoopFuser<REGISTER_COUNT, XARGS...>::registerAction(const char * fileName,
                                     FUSED_SCANVAR[__fusible_global_index__] = SCANVAR[INDEX]; \
                                  } \
                                  }, \
-                              2, __fusible_scan_pos__ , SCANVAR); }
+                              2, __fusible_scan_pos__ , SCANVAR); } FUSIBLE_PHASE_RESET
```

Comment:
I wonder if we need to think out the design a bit more (or perhaps just the naming). It is not intuitive for me to look at this and try to think about how fusible phases are involved here. I think I would rather have another macro like FUSIBLE_FLUSH_POINT or FUSIBLE_FLUSH_IF_NEEDED even if it does the same thing.

Reply:
Yeah, perhaps FUSIBLE_PHASE_RESET could call FUSIBLE_FLUSH_IF_NEEDED?